src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started to work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped to work");

```

Comment:
The message is incorrect. It should be about the Excavator stopping its work, not the Bulldozer. It should be something like 'Excavator stopped working.'